src/distributions/uniform.rs

```diff
@@ -199,7 +199,8 @@ use serde::{Serialize, Deserialize};
 #[cfg_attr(feature = "serde1", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "serde1", serde(bound(serialize = "X::Sampler: Serialize")))]
 #[cfg_attr(feature = "serde1", serde(bound(deserialize = "X::Sampler: Deserialize<'de>")))]
-pub struct Uniform<X: SampleUniform>(X::Sampler);
+// HACK: internals are public for benches
+pub struct Uniform<X: SampleUniform>(pub X::Sampler);
```

Comment:
Can we somehow avoid that this is documented as public?

Reply:
[`#[doc(hidden)]`](https://doc.rust-lang.org/rustdoc/write-documentation/the-doc-attribute.html#hidden) should work

Reply:
That should not have been merged... I think it's no longer required.